### ISSUE TITLE
Suggestion: Group by torrent name

### ISSUE THREAD

**Issue by Kenifornication** [elundmark/tz-aio-userscript#22]
This might sound like a weird suggestion, but here's a hypothetical use case: If you like to search for the most recent torrents (sort by date) in a certain keyword, there bound to be a couple of different torrents for the same item, the difference between the two would be technical but they'd be essentially the same thing, and by grouping (or re-sorting them, if that's a better term) them by torrent name it'd be easier to go through them instead of having them scattered in the results.

TL;DR: Option to sort each page of the search results by name.
